apps/heft/src/schemas/anything.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
What is this needed for?

Reply:
`heft-config-file` requires a schema, and I didn't want to bother putting together a real schema for tsconfig.